apps/pr-deploy-site/pr-deploy-site.js

```diff
@@ -22,62 +22,14 @@ const siteInfo = [
   {
     package: '@fluentui/docs',
     link: './react-northstar',
-    icon: 'FavoriteStar',
+    icon: 'CompassNW',
     title: 'react-northstar',
   },
   {
-    package: '@fluentui/react-avatar',
-    link: './react-avatar/storybook/index.html',
-    icon: 'Contact',
-    title: 'Avatar',
-  },
-  {
-    package: '@fluentui/react-button',
-    link: './react-button/storybook/index.html',
-    icon: 'LikeSolid',
-    title: 'Button',
-  },
-  {
-    package: '@fluentui/react-checkbox',
-    link: './react-checkbox/storybook/index.html',
-    icon: 'CheckboxComposite',
-    title: 'Checkbox',
-  },
-  {
-    package: '@fluentui/react-image',
-    link: './react-image/storybook/index.html',
-    icon: 'FileImage',
-    title: 'Image',
-  },
-  {
-    package: '@fluentui/react-link',
-    link: './react-link/storybook/index.html',
-    icon: 'Link',
-    title: 'Link',
-  },
-  {
-    package: '@fluentui/react-slider',
-    link: './react-slider/storybook/index.html',
-    icon: 'Slider',
-    title: 'Slider',
-  },
-  {
-    package: '@fluentui/react-tabs',
-    link: './react-tabs/storybook/index.html',
-    icon: 'BrowserTab',
-    title: 'Tabs (Pivot)',
-  },
-  {
-    package: '@fluentui/react-text',
-    link: './react-text/storybook/index.html',
-    icon: 'TextOverflow',
-    title: 'Text',
-  },
-  {
-    package: '@fluentui/react-toggle',
-    link: './react-toggle/storybook/index.html',
-    icon: 'ToggleLeft',
-    title: 'Toggle',
+    package: '@fluentui/react-components',
+    link: './react/storybook/index.html',
```

Comment:
this is the same storybook as @fluentui/react ?

Reply:
....oops 🤦‍♀️

Reply:
Okay now it's linking to http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/17289/merge/react-components/storybook/index.html